src/Paysera/Bundle/PhpGeneratorBundle/Resources/config/services/type_configurations.xml

```diff
@@ -137,6 +137,12 @@
                                 </service>
                             </argument>
                         </call>
+                        <call method="setResultPopulatorCode">
+                            <argument>new Money($data['amount'], $data['currency']);</argument>
```

Comment:
As you already see, it is very easy to break things up here.
Please add a test case (RAML and Expected code) that covers your change.